src/components/AttachmentCarousel/index.js

```diff
@@ -197,20 +272,37 @@ class AttachmentCarousel extends React.Component {
                         )}
                     </>
                 )}
-                <CarouselActions
-                    styles={[styles.attachmentModalArrowsContainer]}
-                    canSwipeLeft={!this.state.isBackDisabled}
-                    canSwipeRight={!this.state.isForwardDisabled}
-                    onPress={() => this.canUseTouchScreen && this.toggleArrowsVisibility(!this.state.shouldShowArrow)}
-                    onCycleThroughAttachments={this.cycleThroughAttachments}
-                >
-                    <AttachmentView
-                        onPress={() => this.toggleArrowsVisibility(!this.state.shouldShowArrow)}
```

Comment:
Follow up refactor - the `onPress` prop seem to have been added to the `AttchmentView` and then all the child view that it renders (Image, Pdf, ...) for the arrow functionality 

There are no more usages of AttachmentView with `onPress`, so we can remove it (as well as the other views like ImageView)